src/Form/DropdownField.jsx

```diff
@@ -29,6 +30,12 @@ export const DropdownField = ({
     )
   }, [field, options, optionAttributes, multiple])
 
+  const selectedLabel = multiple
+    ? valueRender
+    : selectedOption
+    ? selectedOption[optionAttributes.label]
+    : ''
+
```

Comment:
What if I want custom valueRender for my single selection dropdownfield?

```suggestion
  const renderSelectedLabel = (value) => {
    // how do we render the label if the consumer did not provide a function??
  }
  const selectedLabel = valueRender || renderSelectedLabel

```

Reply:
Added a fallback in this case 5bbb4e7